compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.test.cpp

```diff
@@ -217,6 +228,27 @@ TEST(RemoveUnnecessaryTransposeNetPass, identity_pattern)
   EXPECT_TRUE(is_transpose_removed(g.g()));
 }
 
+TEST(RemoveUnnecessaryTransposeNetPass, incomplete_reshape_pattern)
```

Comment:
Newly added 3 tests passed.
I also checked our target model(efficient former) is optimized well as we expected.
```sh 
➜ ./build/compiler/luci/pass/luci_pass_test --gtest_filter='RemoveUnnecessaryTransposeNetPass.incomplete*'
Running main() from /home/sseung/Workspace/ONE_second/externals/GTEST/googletest/src/gtest_main.cc
Note: Google Test filter = RemoveUnnecessaryTransposeNetPass.incomplete*
[==========] Running 3 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 3 tests from RemoveUnnecessaryTransposeNetPass
[ RUN      ] RemoveUnnecessaryTransposeNetPass.incomplete_reshape_pattern
[       OK ] RemoveUnnecessaryTransposeNetPass.incomplete_reshape_pattern (0 ms)
[ RUN      ] RemoveUnnecessaryTransposeNetPass.incomplete_reshape_pattern1_NEG
[       OK ] RemoveUnnecessaryTransposeNetPass.incomplete_reshape_pattern1_NEG (0 ms)
[ RUN      ] RemoveUnnecessaryTransposeNetPass.incomplete_reshape_pattern2_NEG
[       OK ] RemoveUnnecessaryTransposeNetPass.incomplete_reshape_pattern2_NEG (0 ms)
[----------] 3 tests from RemoveUnnecessaryTransposeNetPass (0 ms total)

[----------] Global test environment tear-down
[==========] 3 tests from 1 test suite ran. (0 ms total)
[  PASSED  ] 3 tests.
```